package.json

```diff
@@ -15,15 +15,15 @@
     "react-dom": "^17.0.2",
     "react-paginate": "^8.1.3",
     "react-router-dom": "^6.4.2",
-    "react-scripts": "5.0.0",
     "typescript": "^4.8.4",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "BROWSER=none react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
+    "start": "vite",
+    "build": "tsc && vite build",
```

Comment:
Let's change this line to `"build": "tsc --noEmit && vite build",` 

The `--noEmit` part makes it so js files are not added when running `npm run build`.